### ISSUE TITLE
[BUG][8.x] Remove the steps for approving the Endpoint kernel extension 

### ISSUE THREAD

**Issue by nastasha-solomon** [elastic/security-docs#3477]
## Description
- Remove the [Approve the kernel extension](https://www.elastic.co/guide/en/security/8.8/deploy-elastic-endpoint.html#kernel-extension-approval) section (see Ricardo’s [comment](https://elastic.slack.com/archives/CNW9Y42KV/p1686851545485249?thread_ts=1686848250.642509&cid=CNW9Y42KV)) and any references pointing towards that section.
  - Section will only need to be removed from the Install Elastic Endpoint manually on macOS Catalina though Monterey topic, which will become available once https://github.com/elastic/security-docs/pull/3161 is merged. 
- Possibly add a “Deprecations” section in the 7.17 release notes so we can tell users that the Endpoint kernel module was deprecated in 7.17. If we do this, we'll need to create a separate PR for it since we'll need to branch from the 7.17 branch.

## Notes
- Will need to backport this change through all the 8.x doc branches and to the 7.17 branch. Note that in the 7.17 docs, the instructions are in a section titled [Enable Elastic Endpoint kernel](https://www.elastic.co/guide/en/security/7.17/install-endpoint.html#enable-kernel-extension).
- Am making more changes to the Elastic Endpoint install and deploy docs at https://github.com/elastic/security-docs/pull/3161

cc: @Trinity2019 @nfritts @roxana-gheorghe

**Comment by nastasha-solomon** [comment]
Re-opening because I'll need to open a new PR for the 7.17 doc and release note updates.

**Comment by nastasha-solomon** [comment]
Changes for 7.17 branch are being applied in https://github.com/elastic/security-docs/pull/3698.

**Comment by nastasha-solomon** [comment]
Need to check if we should also remove the link to the instructions for enabling the Endpoint kernel on [this page](https://www.elastic.co/guide/en/security/7.17/sensor-full-disk-access.html#macos-permissions) (specifically within in the "Endgame Sensor" and "Elastic Endpoint" sections).

**Comment by nastasha-solomon** [comment]
8.x doc updates merged. Tracking remaining changes for 7.x in https://github.com/elastic/security-docs/issues/3779.

**Comment by nastasha-solomon** [comment]
https://github.com/elastic/security-docs/pull/3698 is merged
